admin_interface/admin.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 
 from django.contrib import admin
-
+from django.utils.translation import ugettext_lazy as _
 from admin_interface.models import Theme
```

Comment:
Minor thing: to fit with the existing style, the blank line between django imports and local project imports could be kept.

Reply:
Done!